### PR TITLE
paper: minor changes for JOSS publication

### PR DESCRIPTION
**Pull request by roth-jakob**

### CODE REVIEW

File: paper/paper.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ Custom gradients can be added to these functions.
 
 Automatic differentiation is a core feature of JAX and often one of the main reasons for using it.
 Thus, it is essential that custom functions registered with JAX support automatic differentiation.
-In the following, we will outline which functions our package respectively JAX requires to enable automatic differentiation.
```
Comment (mreineck) [inline paper/paper.md:81]: I think what's meant is "... which functions our package requires to enable automatic differentiation via JAX".

File: paper/paper.md [hunk 2]
```diff
@@ -176,16 +176,16 @@ In the future, the interface could be easily expanded if specific use cases requ
...
 Expressed in formulas, $\partial f(x)(dx) = f(dx)$ if $f$ is linear in $x$.
-Analogously, the pullback respectively the `vjp` becomes independent of the initial position and is given by the linear transpose of $f$, thus $(\partial f(x))^{T}(dy) = f^T(dy)$.
+Analogously, the pullback or `vjp` becomes independent of the initial position and is given by the linear transpose of $f$, thus $(\partial f(x))^{T}(dy) = f^T(dy)$.
 Also, all higher order derivatives can be expressed in terms of $f$ and its transpose.
 To make use of these simplifications, `JAXbind` provides a special interface for linear functions, supporting higher order derivatives, only requiring an implementation of the function and its transpose.
 
 # Platforms
 
 Currently, `JAXbind` only supports primitives that act on CPU memory.
-In the future, GPU support could be added analogous to the CPU backend.
```
Comment (mreineck) [inline paper/paper.md:188]: How about "In the future, GPU support could be added, which should work analogously to the CPU support in most respects."

The point here is that the effort to get things working on GPU should be much less than it was to produce the initial CPU implementation; it's just that GPU-specific knowledge is required which we don't have.

File: paper/paper.md (hunk 1 above)
Comment (roth-jakob) [reply paper/paper.md:81]: Sounds good to me!

File: paper/paper.md (hunk 2 above)
Comment (roth-jakob) [reply paper/paper.md:188]: Sounds also good to me!

**Comment by roth-jakob** [comment]
@mreineck thanks for your suggestions! I think we could merge this now. What do you think?

**Comment by mreineck** [comment]
Perfectly fine with me!

**Pull request merged**